examples/ticks_and_spines/tick-locators.py

```diff
@@ -66,7 +66,7 @@ def setup(ax):
 # Index Locator
 ax = plt.subplot(n, 1, 5)
 setup(ax)
-ax.plot(range(0, 5), [0]*5, color='white')
+ax.plot(range(0, 5), [0]*5, color='White')
```

Comment:
I would leave basic color words lowercase, even though they are case-insensitive. Seems more natural and common.

https://www.w3.org/TR/css-color-3/ Section 4.1. Basic color keywords

Reply:
done